client/pkg/template/internal/models/machine.go

```diff
@@ -130,14 +131,22 @@ func (machine *Machine) Translate(context TranslateContext) ([]resource.Resource
 	patches, err := machine.Patches.Translate(
 		fmt.Sprintf("cm-%s", machine.Name),
 		constants.PatchBaseWeightClusterMachine,
-		pair.MakePair(omni.LabelCluster, context.ClusterName),
+		pair.MakePair(omni.LabelCluster, ctx.ClusterName),
 		pair.MakePair(omni.LabelClusterMachine, string(machine.Name)),
 	)
 	if err != nil {
 		return nil, err
 	}
 
-	return append(resourceList, patches...), nil
+	resourceList = append(resourceList, patches...)
+
+	schematicConfigurations := machine.SystemExtensions.translate(
+		ctx,
+		string(machine.Name),
+		pair.MakePair(omni.LabelClusterMachine, string(machine.Name)),
```

Comment:
should this also have the cluster label, like the config patches?

Reply:
it already has it. It adds it inside `translate` func

Reply:
oh, I see, all good